chain/cosmos/chain_node.go

```diff
@@ -1008,6 +1068,8 @@ func (tn *ChainNode) CreateNodeContainer(ctx context.Context) error {
 }
 
 func (tn *ChainNode) StartContainer(ctx context.Context) error {
+	rpcOverrideAddr := ""
+
 	for _, s := range tn.Sidecars {
 		err := s.containerLifecycle.Running(ctx)
```

Comment:
If s.preStart is true, err should always be nil. If it is an error, then preStart would have been false (not satisfying the && statement) and everything running as normal. We could add a check to ensure `!s.perStart` but would be kind of duplicated. also don't want a real return here, so it would only log it anyways

*Running() method acts more like a boolean vs error, this logic checks out. Is a little confusing for sure*